packages/teams-js/test/public/runtime.spec.ts

```diff
@@ -36,24 +36,34 @@ describe('runtime', () => {
   describe('runtime versioning', () => {
     it('latestRuntimeVersion should match Runtime interface apiVersion', () => {
       const runtime: Runtime = {
-        apiVersion: 1,
+        apiVersion: 2,
         supports: {},
       };
       expect(latestRuntimeApiVersion).toEqual(runtime.apiVersion);
     });
 
-    it('applyRuntime fast-forwards v0 runtime config to latest version', () => {
-      const runtimeV0 = {
```

Comment:
I assume we will still want to support version 0 and upgrading of 0 to 1, so we should still test that.

Reply:
Talked with Shreyas, v0 was just for testing and is safe to remove